next.config.js

```diff
@@ -61,7 +59,7 @@ module.exports = withSentryConfig(
     transpileClientSDK: true,
 
     // Routes browser requests to Sentry through a Next.js rewrite to circumvent ad-blockers (increases server load)
-    tunnelRoute: "/monitoring",
+    tunnelRoute: '/monitoring',
```

Comment:
What's this for @oxkatsudon ?

Reply:
🤷 
https://github.com/getsentry/sentry-wizard

Reply:
What does increase server load mean? Is this applicable without a backend?

Reply:
I guess it auto-falls back to not tunneling if it doesn't succeed if things are working and we have no backend? Or maybe it works on Vercel but not on prod/IPFS?